lib/fathead/mdn_css/parse.pl

```diff
@@ -237,11 +223,25 @@ sub make_and_write_article {
     say "LINK: $link";
     say "DESCRIPTION: $description"       if $description;
     say "EXTERNAL LINKS $external_links " if $external_links;
-    my $data = join "\t", (
+    my @data = join "\t", (
         $title, 'A', '', '', '', '', '', '', $external_links || '',
         '', '', $description, $link
     );
-    write_to_file($data);
+
+    # Check for CSS Functions
+    # e.g. "not()"
+    # Replace "()" with "function" in title
+    # e.g. "not()" - > "not function"
+    if ( $title =~ m/\(\)$/ ) {
```

Comment:
We may want to pull this into a new `sub` so we can add additional logic for creating redirects.

Should be okay for now though 👍